README.md

```diff
@@ -141,31 +143,27 @@ However, we strongly recommend that you use a real device for development, as so
 
 ## Android
 
-**3.** Install necessary rust targets (this set may vary depending on the target architecture
-   you are building for be it android studio emulators or hardware devices):
+**4.** Install necessary rust targets (this set may vary depending on the target device architecture
+   you are building for):
 
    ```bash
     rustup target add aarch64-linux-android armv7-linux-androideabi x86_64-linux-android
    ```
+Note - old x86 is not supported. Just use x86_64 emulator image.
 
-**4.** Download [Android Studio](https://developer.android.com/studio).
-
-**5.** Open the project from the `android` directory.
+**5.** Download [Android Studio](https://developer.android.com/studio).
 
-**6.** Install NDK. Go to `File -> Project Structure -> SDK Location`. Next to the "Android NDK location" section, click "Download Android NDK" button.
+**6.** Open the project from the `android` directory.
 
-⚠️  We highly recommend you to update all existing plugins and SDK's for Kotlin, Gradle,
-etc even if you just downloaded a fresh Android Studio. It's always a good idea to restart
-Android Studio after that. This can save you many hours on Stackoverflow trying to fix
-random errors like "NDK not found".
+**7.** Install NDK. Currently specific version 24.0.8215888 is required.
```

Comment:
Please add information about how to install NDK

Reply:
done